install_config/install/advanced_install.adoc

```diff
@@ -680,6 +680,217 @@ openshift_examples_modify_imagestreams=true
+using GlusterFS.
+endif::[]
+
+The following procedure installs all of the required components,
+including `Endpoints`, `Services`, and a `StorageClass`.
+
+[IMPORTANT]
+====
+See
+xref:../../install_config/install/prerequisites.adoc#prereq-containerized-glusterfs-considerations[Containerized
+GlusterFS Considerations] for specific host preparations and prerequisites.
+====
+
+. In your inventory file, add `glusterfs` to the `OSEv3:children` section to enable the `[glusterfs]` group:
++
+----
+[OSEv3:children]
+masters
+nodes
+glusterfs
+----
+
+. (Optional) Include any of the following role variables in the `[OSEv3:vars]`
+section you wish to change:
++
+----
+[OSEv3:vars]
+openshift_storage_glusterfs_namespace=glusterfs <1>
+openshift_storage_glusterfs_name=storage <2>
+----
+<1> The project (namespace) to host the storage pods. Defaults to `glusterfs`.
+<2> A name to identify the GlusterFS cluster, which will be used in resource names.
+Defaults to `storage`.
+
+. Add a `[glusterfs]` section with entries for each storage node that will host
+the GlusterFS storage and include tnclude the `glusterfs_ip` and
+`glusterfs_devices` parameters in the form:
++
+----
+<hostname_or_ip> glusterfs_ip=<ip_address> glusterfs_devices='[ "</path/to/device1/>", "</path/to/device2>", ... ]'
+----
++
+For example:
++
+----
+[glusterfs] 
+192.168.10.11 glusterfs_ip=192.168.10.11 glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+192.168.10.12 glusterfs_ip=192.168.10.12 glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+192.168.10.13 glusterfs_ip=192.168.10.13 glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
+----
++
+Set `glusterfs_devices` to a list of raw block devices that will be completely
+managed as part of a GlusterFS cluster. There must be at least one device
+listed. Each device must be bare, with no partitions or LVM PVs. Set
+`glusterfs_ip` to the IP address that will be used by pods to communicate with
+the GlusterFS node.
+
+. After completing the cluster installation per
+xref:running-the-advanced-installation[Running the Advanced Installation], run
+the following from a master to verify the necessary objects were successfully
+created:
+
+.. Verfiy that the GlusterFS `StorageClass` was created:
++
+----
+# oc get storageclass
+NAME                  TYPE
+glusterfs-storage     kubernetes.io/glusterfs
+----
+
+.. Verify that the route was created:
++
+----
+# oc get routes
+NAME            HOST/PORT                                     PATH           SERVICES   PORT   TERMINATION   WILDCARD
+heketi-glusterfs-route  heketi-glusterfs-default.cloudapps.example.com  heketi-glusterfs <all>             None
+----
++
+[NOTE]
+====
+The name for the route will be `heketi-glusterfs-route` unless the default
+`glusterfs` value was overridden using the `openshift_glusterfs_storage_name`
+variable in the inventory file.
```

Comment:
s/overridden/overwritten ?
(Overridden sounds more like what a governor does to a bill from the legislature)